Chummer/Backend/Uniques/Spell.cs

```diff
@@ -102,6 +103,11 @@ public void Create(XmlNode objXmlSpellNode, string strForcedValue = "", bool bln
             }
             if (!objXmlSpellNode.TryGetMultiLineStringFieldQuickly("altnotes", ref _strNotes))
                 objXmlSpellNode.TryGetMultiLineStringFieldQuickly("notes", ref _strNotes);
+
+            String sNotesColor = ColorTranslator.ToHtml(ColorManager.HasNotesColor);
+            objXmlSpellNode.TryGetStringFieldQuickly("notesColor", ref sNotesColor);
+            _colNotes = ColorTranslator.FromHtml(sNotesColor);
+
```

Comment:
Is this necessary in the `Create` method? Would there need to be data-definable colors for specific spells?

Reply:
Let's use this one to discuss also the other occurences.

Initially I was only trying to achive the same behaviour as with notes themselfs. Notes can occur in a data file and therefore I added the color section, too.

After a bit of consideration, I could imagine a real use case for this in the data-files. Especially when adding notes in data files, chosing a color that matches it's meaning (warning=yellow, default note = blue, ...) 

Personally I don't really care if we have it in here or not. If you like I can also remove it and it can be added later if a (better) use-case comes up.